### ISSUE TITLE
cargo-asm prints color codes even when stdout is not a terminal

### ISSUE THREAD

**Issue by bheisler** [gnzlbg/cargo-asm#197]
When I try to run `cargo asm <args> | less` I see a bunch of escape codes in less. It would be nice if cargo-asm could detect if stdout is a terminal, or at least provide a `--color=never` option.

**Comment by malaire** [comment]
I don't think this is a bug as you might want color codes when stdout is not terminal, e.g. to save to file which you then open in editor which supports color codes.

Also there is already `--no-color` to not output colors.
